src/exchanges/fetch.ts

```diff
@@ -146,3 +151,24 @@ const executeFetch = (
       }
     });
 };
+
+export const convertToGet = (uri: string, body: Body): string => {
+  const queryParams: string[] = [];
+  if (body.query) {
+    queryParams.push(`query=${encodeURIComponent(body.query)}`);
+  }
+
+  if (body.operationName) {
+    queryParams.push(`operationName=${encodeURIComponent(body.query)}`);
+  }
+
+  if (body.variables) {
+    queryParams.push(
+      `variables=${encodeURIComponent(JSON.stringify(body.variables))}`
+    );
+  }
+
+  // TODO: fragments
```

Comment:
Not sure what this means. Fragments are part of the document and shouldn’t need any special logic

Reply:
I thought there was a complex issue with this on a previous thing I made. Let's see when I test it